buddyboss/buddyboss-platform
# Terms & Condition and Privacy Policy page returns 404 page.

### All Submissions:






### Changes proposed in this Pull Request:
Terms & Condition and Privacy Policy page returns 404 page with "Instructor Role" by WisdmLabs and "Uncanny LearnDash Toolkit Pro"

Closes #470.

### How to test the changes in this Pull Request:

1. Go to Admin Dashboard'
2. Activate one of the above-listed plugin.
3. Create Terms & Conditions and Privacy Policy page from Pages >> Add New Section
4. Now assign both the pages inside buddyboss >> Pages section
5. Click on the view page button and see the error.

### Proof Screenshots or Video
http://somup.com/cYV0htfRSr


### Other information:
